examples/todo/src/openapi-spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {OpenAPIObject} from '@loopback/rest';
+import {TodoListApplication} from './application';
+
+/**
+ * Export the OpenAPI spec from the application
+ */
+async function exportOpenApiSpec(): Promise<OpenAPIObject> {
+  const config: ApplicationConfig = {
+    rest: {
+      port: +(process.env.PORT ?? 3000),
+      host: process.env.HOST ?? 'localhost',
+      openApiSpec: {
+        setServersFromRequest: true,
```

Comment:
AFAIK, `setServersFromRequest` is used only when the OpenAPI spec is requested via HTTP. Since we are calling `getApiSpec` directly, I believe the `servers` field will be always populated using `host` and `port` values from the config, and thus the entire `openApiSpec` section can be removed from the config here.

Reply:
Cleaned.